cypress/e2e/listingDetails/completedLotteryListing.e2e.ts

```diff
@@ -1,13 +1,5 @@
-const listings = {
-  COMPLETED_LOTTERY_FULL: {
-    id: "a0W8H000000AmpKUAS",
-    lotteryNumber: "01150047",
-  },
-  COMPLETED_LOTTER_PROD: {
-    id: "a0W4U00000IhGZcUAN",
-    lotteryNumber: "01171676",
-  },
-}
+const listingId = "a0W8H000000AmpKUAS"
+const lotteryNumber = "01150047"
```

Comment:
question: Are these the intended values? They are the full environment values, and probably won't work when running prod tests.

Reply:
Prod tests will use fixtures